api/controllers/tokens.js

```diff
@@ -0,0 +1,27 @@
+const User = require("../models/user");
+const TokenGenerator = require("../models/token_generator");
+const bcrypt = require("bcrypt");
+
+const SessionsController = {
+  Create: (req, res) => {
+    const email = req.body.email;
+    const password = req.body.password;
+
+    User.findOne({ email: email }).then(async (user) => {
+      const isPasswordValid = await bcrypt.compare(password, user.password);
+      if (!user) {
+        console.log("auth error: user not found");
+        res.status(401).json({ message: "auth error" });
+      } else if (!isPasswordValid) {
+        console.log("auth error: passwords do not match");
+        res.status(401).json({ message: "auth error" });
```

Comment:
Message could be a little more specific for the user in the console of the web engine.
Instead of 'auth error'.
It could be ' Password does not match'